sysdeps/linux/include-internal/linux/unistd.h

```diff
@@ -0,0 +1 @@
+#include <syscallnos.h>
```

Comment:
Can't this live in `include/bits`?

Reply:
it's not installed

Reply:
Can't we just not install it then? bc having both `include/bits` and `include-internal` seems a bit unnecessary to me